pkg/obs/nodeserver.go

```diff
@@ -93,22 +94,17 @@ func (ns *nodeServer) NodePublishVolume(_ context.Context, req *csi.NodePublishV
 		return nil, status.Errorf(codes.Internal, "Failed to make dir: %s, error: %v", targetPath, err)
 	}
 
-	credentialFile := fmt.Sprintf("%s/%s", credentialDir, uuid.New().String())
+	credentialFile := filepath.Join(credentialDir, uuid.New().String(), credentialFileName)
 	accessKey := ns.Driver.cloud.Global.AccessKey
 	secretKey := ns.Driver.cloud.Global.SecretKey
 	if err := createCredentialFile(accessKey, secretKey, credentialFile); err != nil {
 		return nil, err
 	}
 	defer deleteCredentialFile(credentialFile)
 
-	mountFlags := []string{"big_writes", "max_write=131072", "use_ino"}
+	mountFlags := []string{"big_writes", "max_write=131072", "nonempty"}
 	if mnt := req.GetVolumeCapability().GetMount(); mnt != nil {
-		for _, v := range mnt.GetMountFlags() {
```

Comment:
Specifying the value of `passwd_file` is not allowed.

Reply:
If possible, validation should be added to avoid duplication of customer-specified configuration with the default configuration.

Reply:
1. `pwd_file` is created by `echo 'AK:SK' > /xxx/.pwd`，it is difficult and unnecessary to find it
2.  If one use CSI OBS plugin to create bucket, the config should be consistent with `cloud-config` rather than other configuration